lawnchair/src/app/lawnchair/ui/preferences/components/colorpreference/ColorDot.kt

```diff
@@ -17,27 +21,53 @@ fun <T> ColorDot(
     entry: ColorPreferenceEntry<T>,
     modifier: Modifier = Modifier
 ) {
+    val context = LocalContext.current
+
+    val colorLight = entry.lightColor(context)
+    val colorDark = entry.darkColor(context)
```

Comment:
Unnecessary change?

Reply:
I created the variables because I needed to use `colorLight` on line 35.